features/FxA-54-push-on-status-change/README.md

```diff
@@ -0,0 +1,84 @@
+# Push Notifications for Account Status Changes
+
+https://mozilla.aha.io/features/FXA-54
+
+## Stories
+
+As a user signed in to an FxA connected device,
+I want my device to update its UI in a timely manner
+when I interact with my account from other devices.
+
+As a developer of client code for a connected device,
+I want to easily receive notifications so that I can
+respond to account status changes without resorting to
+polling the server.
+
+As a developer of Firefox Accounts,
+I want to stop Firefox from polling for account verification
+because this is the single biggest source
+of system load in production.
+
+## Success Criteria
+
+This feature will be successful if
+we successfully eliminate polling for account verification
+from the browser code on Desktop and Fennec.
+
+As a concrete metric
+we can measure the time between account verification
+and the use of a keyFetchToken by the browser,
+an action that it will only perform
+when it knows the account has been verified.
```

Comment:
What I was going for here is some sort of "timeliness" metric, a way to measure whether the browser is proceeding to sync more quickly because we got the push right away, rather than having to wait for the polling to complete.  Checking server load to confirm that "we're polling the server less" tells us the success of the developer-focused  story, but not success of the user-focused story.

But, it's possible that difference will only be on the order of a second or two anyway, so hard to measure effectively.